kotlin-vercel-ncc/src/jsMain/kotlin/vercel/AssetMap.kt

```diff
@@ -0,0 +1,21 @@
+package vercel
+
+import js.objects.Object
+import kotlinx.js.JsPlainObject
+
+@JsPlainObject
+external interface AssetMap
```

Comment:
Looks like `Record<String, Asset>`

Reply:
Indeed. How do I iterate over a `Record`'s entries?
With the `AssetMap` I had 
```kotlin
result.assets?.forEach { (assetFileName, asset) -> ... }
```
With the `Record` is it then
```kotlin
for ((assetFileName, asset) in entries(result.assets ?: recordOf())) { ... }
```
?

Reply:
`Object.entries()` will give you required iterable.
And it works for any type, not only for `Record`

Reply:
So what I did, yeah?
Just - as I just have seen myself -
```kotlin
entries(result.assets ?: recordOf()).forEach { (assetFileName, asset) -> ... }
```
or
```kotlin
result.assets?.let(Object::entries)?.forEach { (assetFileName, asset) -> ... }
```
👌

Reply:
```kotlin
for ((assetFileName, asset) in Object.entries(result.assets)) { ... }
```
should work